system/HTTP/Request.php

```diff
@@ -12,11 +12,12 @@
 namespace CodeIgniter\HTTP;
 
 use CodeIgniter\Validation\FormatRules;
+use Config\App;
 
 /**
- * Representation of an HTTP request.
+ * Representation of an incoming, server-side HTTP request.
  */
-class Request extends Message implements RequestInterface
+class Request extends OutgoingRequest implements RequestInterface
```

Comment:
The comment and the extension seem in opposition. Is it incoming or outgoing?

Reply:
(incoming)Request extends OutgoingRequest.

Reply:
Right! We cleared this up in the other threads, I never came back and closed this. Not an issue!